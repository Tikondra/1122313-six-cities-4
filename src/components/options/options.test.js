import React from "react";
import renderer from "react-test-renderer";
import Options from "./options";

it(`should render Options`, function () {
  const tree = renderer
    .create(
        <Options options={[`Wi-Fi`, `Heating`, `Kitchen`, `Fridge`]}/>
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
