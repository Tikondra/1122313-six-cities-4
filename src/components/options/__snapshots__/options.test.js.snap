// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Options 1`] = `
<div
  className="property__inside"
>
  <h2
    className="property__inside-title"
  >
    What's inside
  </h2>
  <ul
    className="property__inside-list"
  >
    <li
      className="property__inside-item"
    >
      Wi-Fi
    </li>
    <li
      className="property__inside-item"
    >
      Heating
    </li>
    <li
      className="property__inside-item"
    >
      Kitchen
    </li>
    <li
      className="property__inside-item"
    >
      Fridge
    </li>
  </ul>
</div>
`;
